src/test/java/uk/gov/hmcts/probate/services/persistence/services/SequenceNumberServiceTest.java

```diff
@@ -6,50 +6,141 @@
 import org.mockito.runners.MockitoJUnitRunner;
 import org.springframework.jdbc.support.incrementer.PostgreSQLSequenceMaxValueIncrementer;
+import uk.gov.hmcts.probate.services.persistence.TestUtils;
+import uk.gov.hmcts.probate.services.persistence.model.Registry;
 import uk.gov.hmcts.probate.services.persistence.model.RegistryNotConfiguredException;
+import uk.gov.hmcts.probate.services.persistence.repository.RegistryRepository;
+
+import javax.persistence.EntityManager;
 
 @RunWith(MockitoJUnitRunner.class)
 public class SequenceNumberServiceTest {
 
-  @Rule
-  public ExpectedException expectedException = ExpectedException.none();
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @Mock
+    private Map<String, PostgreSQLSequenceMaxValueIncrementer> registrySequenceNumbers;
+
+    @Mock
+    private PostgreSQLSequenceMaxValueIncrementer mockIncrementer;
+
+    @Mock
+    private RegistryRepository registryRepository;
+    @Mock
+    private ObjectMapper mapper;
+    @Mock
+    private EntityManager entityManager;
+
+    private TestUtils testUtils;
+    private SequenceNumberService sequenceNumberService;
+    private List<Registry> registryList;
+
+    private Registry oxford = new Registry();
+    private Registry birmingham = new Registry();
+    private Registry manchester = new Registry();
+
+
+    @Before
+    public void setUp() {
+        oxford.setId("oxford");
+        oxford.setEmail("oxford@email.com");
+        oxford.setAddress("Test Address Line 1\nTest Address Line 2\nTest Address Postcode");
+        oxford.setRatio(3L);
+        oxford.setCounter(2L);
+        birmingham.setId("birmingham");
+        birmingham.setEmail("birmingham@email.com");
+        birmingham.setAddress("Birmingham Test Address Line 1\nTest Address Line 2\nTest Address Postcode");
+        birmingham.setRatio(3L);
+        birmingham.setCounter(3L);
+        manchester.setId("manchester");
+        manchester.setEmail("manchester@email.com");
+        manchester.setAddress("Manchester Test Address Line 1\nTest Address Line 2\nTest Address Postcode");
+        manchester.setRatio(3L);
+        manchester.setCounter(3L);
+        registryList = new ArrayList<>();
+        registryList.add(oxford);
+        registryList.add(birmingham);
+        registryList.add(manchester);
+        testUtils = new TestUtils();
+        mapper = new ObjectMapper();
+        sequenceNumberService = new SequenceNumberService(registrySequenceNumbers, registryRepository, entityManager, mapper);
+    }
+
+    @Test
+    public void getNext() {
+        when(this.registrySequenceNumbers.get("oxford"))
+            .thenReturn(mockIncrementer);
+        when(this.mockIncrementer.nextLongValue())
+            .thenReturn(1234L);
+
+        Long response = sequenceNumberService.getNextSequenceNumber("oxford");
+
+        assertThat(response, is(equalTo(1234L)));
+    }
+
+    @Test
+    public void getNextInValidName() {
```

Comment:
Minor point - it's only a test but 'Invalid' in one word so doesn't need the caps on 'V'.